bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -556,19 +556,32 @@ class PlotView extends Renderer.View
     # user-configurable in the future, as it may not be the right number
     # if people set a large border on their plots, for example.
 
-    # We need the parent node, because the el node itself collapses to zero
-    # height. It might not be available though, if the initial resize
+    # Try to find bk-root. We will query the parent of that node for its size
+    node = @.el
+    for i in [0..2]  # Use for-loop; if we ever change DOM structure, it should still work
+      if node is null or node.classList.contains('bk-root')
+         break
+      node = node.parentNode
```

Comment:
That makes sense. I am not very familiar with jquery so I tend to write more vanilla js. What does that do exactly? I deliberately only search 3 levels up, so that subplots are easily ruled out.